### ISSUE TITLE
Allow order with quantity

### ISSUE THREAD

**Issue by riderx** [j3k0/cordova-plugin-purchase#1450]
### Observed behaviour

I can see in the code the plugin allow only to order one by one, in my company we need to be able to order multiple consumable at once, can that be added ?

**Comment by TobiasGrether** [comment]
Yeah I second this. I'm not sure if I just don't see this or if it really isn't available yet

**Comment by sirbarkley** [comment]
Anyone got the answer to this?

**Comment by riderx** [comment]
I created a plugin for capacitor to allows that
